src/main/resources/reference.conf

```diff
@@ -4,6 +4,7 @@
 conseil: {
   hostname: "0.0.0.0"
   port: 1337
+  cache-ttl: 15 minutes
```

Comment:
Shouldn't we have a specific timeout for each operation rather than one global timeout?

Reply:
actually we have only 2 endpoints using timeout, so I don't feel that it is necessary to have different TTLs for those endpoints -  but this could change when I get feedback about real usage, now its like fortune telling if this value will be ok, or if we will need different timeouts for different endpoints